src/RoomsSDKAdapter.integration.test.js

```diff
@@ -1,42 +1,45 @@
 import {createIntegrationTestUser, removeIntegrationTestUser} from './testHelper';
-import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+import WebexSDKAdapter from './';
 
 describe('Rooms SDK Adapter', () => {
-  let roomsSDKAdapter, subscription, user;
+  let createdRoom, getRoom$, webexSDKAdapter, subscription, user;
 
   // Since these are integration tests with live data,
   // increase the async "idle" timeout so jest doesn't error early.
   jest.setTimeout(30000);
 
   beforeAll(async () => {
     user = await createIntegrationTestUser();
-  });
-
-  beforeEach(() => {
-    roomsSDKAdapter = new RoomsSDKAdapter(user.sdk);
+    webexSDKAdapter = new WebexSDKAdapter(user.sdk);
+    await webexSDKAdapter.connect();
   });
 
   afterAll(async () => {
-    // This is done after unsubscribing, but since unsubscribe can't be async, manually call it.
-    await user.sdk.rooms.stopListening();
-
     try {
       await removeIntegrationTestUser(user);
+      await webexSDKAdapter.disconnect();
     } catch (reason) {
       // eslint-disable-next-line no-console
       console.warn('Failed to delete test user for Room SDK Adapter integration tests', reason);
     }
   });
 
   describe('getRoom() returns', () => {
-    afterEach(() => {
+    beforeEach(async () => {
+      createdRoom = await user.sdk.rooms.create({title: 'Webex Teams Test Room'});
+      getRoom$ = webexSDKAdapter.roomsAdapter.getRoom(createdRoom.id);
+    });
+
+    afterEach(async () => {
+      await user.sdk.rooms.remove(createdRoom.id);
+      getRoom$ = null;
```

Comment:
Still think that setting these to null is extraneous since the jest test runner should be doing this for us anyway.

Reply:
sure. I can remove them both.

Reply:
Do you have documentation for this?
It can be extraneous in some cases, like this one but it might matter in others, like re-using `createdRoom` that may point to a removed room

Reply:
For the record, @akoushke and I agreed that we will `null` the variables, but will send a warming when something failed to clean up. This way we won't be masking any potential issues, but not crashing the test run either